strawberryfields/backends/states.py

```diff
@@ -25,11 +25,11 @@
 from scipy.stats import multivariate_normal
 from scipy.special import factorial
 from scipy.integrate import simps
-from thewalrus.quantum import photon_number_mean, photon_number_covar
+#from thewalrus.quantum import photon_number_mean, photon_number_covar
```

Comment:
@Lucaman99, how come these are commented out?

Reply:
Yep, sorry, I meant to un-comment these! For some reason when I tried to test the code, these imports weren't working, but it's probably a problem with my local copy of my fork.

Reply:
🤔 You may have to upgrade your version of the walrus: `pip install thewalrus --upgrade`

Reply:
Thanks, I'll try updating it 🙂